macaroonbakery/store.py

```diff
@@ -10,13 +10,13 @@ class MemoryOpsStore:
     def __init__(self):
         self._store = {}
 
-    def put_ops(self, key, ops, time):
+    def put_ops(self, ctx, key, ops, time):
```

Comment:
As mentioned in Oven, I'm not sure that the context is appropriate here.

Reply:
done.